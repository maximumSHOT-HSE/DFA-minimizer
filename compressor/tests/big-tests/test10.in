13 78 6
1 4 0
1 5 1
1 9 2
1 1 3
1 13 4
1 7 5
2 3 0
2 1 1
2 8 2
2 13 3
2 7 4
2 13 5
3 8 0
3 2 1
3 5 2
3 6 3
3 10 4
3 13 5
4 2 0
4 3 1
4 11 2
4 6 3
4 13 4
4 2 5
5 2 0
5 5 1
5 12 2
5 9 3
5 8 4
5 5 5
6 10 0
6 1 1
6 10 2
6 3 3
6 10 4
6 1 5
7 9 0
7 4 1
7 3 2
7 10 3
7 10 4
7 2 5
8 1 0
8 12 1
8 13 2
8 10 3
8 11 4
8 3 5
9 8 0
9 4 1
9 4 2
9 10 3
9 8 4
9 5 5
10 11 0
10 10 1
10 13 2
10 4 3
10 5 4
10 13 5
11 2 0
11 6 1
11 10 2
11 12 3
11 10 4
11 11 5
12 8 0
12 13 1
12 7 2
12 3 3
12 6 4
12 13 5
13 5 0
13 4 1
13 10 2
13 11 3
13 7 4
13 13 5
3
13
11 12 9 13 1 12 11 3 2 4 10 9 5 
