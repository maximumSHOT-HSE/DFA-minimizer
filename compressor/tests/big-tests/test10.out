13 78 6
8 1 0
2 1 1
6 1 1
1 1 3
6 1 5
4 2 0
5 2 0
11 2 0
3 2 1
4 2 5
7 2 5
2 3 0
4 3 1
7 3 2
6 3 3
12 3 3
8 3 5
1 4 0
7 4 1
9 4 1
13 4 1
9 4 2
10 4 3
13 5 0
1 5 1
5 5 1
3 5 2
10 5 4
5 5 5
9 5 5
11 6 1
3 6 3
4 6 3
12 6 4
12 7 2
2 7 4
13 7 4
1 7 5
3 8 0
9 8 0
12 8 0
2 8 2
5 8 4
9 8 4
7 9 0
1 9 2
5 9 3
6 10 0
10 10 1
6 10 2
11 10 2
13 10 2
7 10 3
8 10 3
9 10 3
3 10 4
6 10 4
7 10 4
11 10 4
10 11 0
4 11 2
13 11 3
8 11 4
11 11 5
8 12 1
5 12 2
11 12 3
12 13 1
8 13 2
10 13 2
2 13 3
1 13 4
4 13 4
2 13 5
3 13 5
10 13 5
12 13 5
13 13 5
3
10
1 2 3 4 5 9 10 11 12 13 
